opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -203,6 +204,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
     { name: 'password_policy_min_words', label: 'Min policy words', type: 'numeric', precision: 'integer', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: false },
     { name: 'password_policy_min_lowercase', label: 'Min policy lowercase', type: 'numeric', precision: 'integer', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: false },
     { name: 'password_policy_min_uppercase', label: 'Min policy uppercase', type: 'numeric', precision: 'integer', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: false },
+    { name: 'default_group_id_for_ingestion_users', label: 'Default groupId for ingestion users', type: 'string', format: 'text', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: false, featureFlag: CSV_FEED_FEATURE_FLAG },
```

Comment:
I'm not sure for the data model because we will have several default group for several screen. Let me think about it.